cookbooks/symfony_no_flex.rst

```diff
@@ -0,0 +1,300 @@
+SymfonyCloud without Flex
+=========================
+
+SymfonyCloud experience is optimal when Symfony applications are using Flex, but
+not every project has been migrated to Flex yet. Even if you are using Symfony
+2.x or 3.x, you can start using SymfonyCloud and actually leverage some of its
+features to ease the migration to Symfony 4 and Flex.
+
+.. tip::
+
+   Although this guide helps you setuping your own project for SymfonyCloud, we
+   suggest you first follow the :doc:`Getting Started </getting-started>`
+   guide with a brand new Symfony 4 project and a free trial to get a better
+   understanding of SymfonyCloud features and philosophy.
+
+Initial setup
+-------------
+
+.. note::
+
+   You have to :ref:`install the CLI Tool <cli-install>` first.
+
+From within your project's directory, generate a sensible default SymfonyCloud
+configuration:
+
+.. code-block:: terminal
+
+    $ symfony project:init
+
+Thanks to our versatile templates most of the hard work of adapting the
+configuration to your Symfony project structure is already done
+for you in the ``.symfony.cloud.yaml`` and
+``app/config/parameters_symfonycloud.php`` files.
```

Comment:
I would probably add a note here about this PHP file, which is where we cope with the missing features from Flex.